config/src/config.rs

```diff
@@ -510,6 +510,10 @@ pub struct Config {
     #[dynamic(default)]
     pub macos_window_background_blur: i64,
 
+    /// Only works on Windows
+    #[dynamic(default)]
+    pub windows_system_backdrop: SystemBackdrop,
```

Comment:
I'd lean towards calling this win32 rather than windows to disambiguate from generic non-OS windows options.

```suggestion
    pub win32_system_backdrop: SystemBackdrop,
```